api/openapi.yml

```diff
@@ -2982,7 +2997,14 @@ components:
           $ref: '#/components/schemas/BlockCurrencyCollection'
         minted:
           $ref: '#/components/schemas/BlockCurrencyCollection'
-
+    IndexingLatency:
+      type: object
+      required:
+        - indexing_latency
+      properties:
+        indexing_latency:
+          type: integer
+          example: 100
```

Comment:
add field 
api_online: bool
and return false if database or client connections returns error